subworkflows/local/analyse_MT.nf

```diff
@@ -63,7 +63,7 @@ workflow ANALYSE_MT {
             genome_dict,
             shift_mt_backchain,
             genome_fasta_no_meta)
-        ch_versions = ch_versions.mix(PICARD_LIFTOVERVCF.out.versions)
+        ch_versions = ch_versions.mix(PICARD_LIFTOVERVCF.out.versions.first())
```

Comment:
I haven't used `first()` in other places in this subworkflow because I have used them inside those specific subworkflow. So for any given process, the version information is obtained only once.